Simplify and Clarify configs Spreadsheet Documentation (#247)
commerce-docs/microsite-commerce-storefront: src/content/docs/setup/analytics/instrumentation.mdx

The Commerce boilerplate includes the ACDL by default.
:::

## Configuration

> To configure environment variables for your Commerce backend, you must update the `analytics-config` spreadsheet located in the root of your SharePoint content directory. Open the file to view the connection **keys** and **values** of the sample backend. They should look similar (but not exact) to the following:

Reviewer: ```suggestion
To configure environment variables for your Commerce backend, you must update the `analytics-config` spreadsheet located in the root of your SharePoint content directory. Open the file to view the connection **keys** and **values** of the sample backend, which should resemble the following:
```
